app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,237 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.NewTask;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskService 클래스")
+class TaskServiceTest {
+
+    private TaskService taskService;
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+    }
+
+    @Nested
+    @DisplayName("getTasks 메소드는")
+    class Describe_getTasks {
+        List<Task> subject() {
+            return taskService.getTasks();
+        }
+
+        @Nested
+        @DisplayName("DB에 데이터가 없는데 조회를 한다면")
```

Comment:
이렇게 해 봅시다. 만약 `getTasks`의 기능이 조회가 아니라 다른 것으로 변경된다면 이 설명도 변경되어야겠죠. 즉, 컨텍스트(환경)에 대한 설명이 메소드의 기능에 의존하지 않는 방향으로 설명을 작성해 보세요.
```suggestion
        @DisplayName("DB에 데이터가 있다면")
```

Reply:
조언 감사합니다. 참고해서  수정해보겠습니다. 😀